### DISCUSSION TITLE
Autocomplete, Dropdown, Select with dymamic options using Schema. Is it possible? how? Please extend doc section

### DISCUSSION THREAD

**Discussion by VladimirFrunza** [formkit/formkit#1150] [General]
How to create Autocomplete, Dropdown, Select with dymamic options using Schema. Is it possible? how? Please extend doc section?

**Comment by fenilli** [comment]
There is already docs for dynamic options: https://formkit.com/inputs/autocomplete#dynamic-options

The schema is the way schema works with functions from https://formkit.com/essentials/schema#references

```js
const schema = {
  $formkit: 'autocomplete',
  options: '$searchMovies'
}

const data = {
  searchMovies: ({ search }) => {
    console.log(search)
    return []
  }
}
```
